src/main/java/calculator/exception/BlankCheckException.java

```diff
@@ -0,0 +1,8 @@
+package calculator.exception;
+
+public class BlankCheckException extends RuntimeException{
+  private static final String MESSAGE ="null 또는 공백을 입력하실 수 없습니다.";
+  public BlankCheckException() {
+    super(MESSAGE);
+  }
```

Comment:
- 사용자 정의 예외 활용 👍
- 여기도 가독성을 위해서 공백 라인을 추가해보면 좋을 것 같습니다.
```suggestion
  private static final String MESSAGE = "null 또는 공백을 입력하실 수 없습니다.";
  
  public BlankCheckException() {
    super(MESSAGE);
  }
```